### ISSUE TITLE
Prores with Alpha

### ISSUE THREAD

**Issue by hansnolte** [Rainbox-dev/DuME#242]
Hi Nico,

with the original settings of DuME 0.3 only ProresHQ works.
I downloaded the presets from here and they work.
But I want to have an alpha channel, so "Add Settings"->Adjust Transparency/Alpha".
If I now want to export Prores (tested with all presets) with an alpha channel, all files are saved with 0 bytes.

Did I do something wrong or forget something?
Or is that a bug?

I am on Windows 10 - latest Nvidia Studiodriver and a GTX 1070.
The input Video for my test was a Quicktime PNG Animation with transparent Background.

**Comment by Nico-Duduf** [comment]
Thanks for the feedback
I'll run some tests and let you know

**Comment by Nico-Duduf** [comment]
Actually, only ProRes 4444 and 4444 XQ support alpha channels, and I don't think DuME can support these ProRes codecs, it may support only up to HQ

I'll confirm that ASAP

**Comment by hansnolte** [comment]
Hi Nico,

many thanks for your fast feedback.

Why should DuME not support 4444, FFMPEG does?

https://trac.ffmpeg.org/wiki/Encode/VFX
_" Prores is a 422 codec, with an existing 4444 variation. FFmpeg comes with 3 different prores encodes: "prores", "prores_ks" (formerly named "prores_kostya") and "prores_aw" (formerly named "prores_anatolyi"). In our testing we've used the "prores" and the "prores_ks" encoders and found "prores_ks" to be the best encoder to use. It is the only one that supports the 4444 colorspace and although it may be slightly slower. The color quality of the videos produced by these two codecs was visually indistinguishable Because of the 4444 support we've decided to go with Kostya's version of prores.

There are 4 profiles that exist within Prores: Proxy, LT, SQ and HQ (and then optionally 4444). In ffmpeg these profiles are assigned numbers (0 is Proxy and 3 is HQ). See Apple's official Prores whitepaper for details on the codec and information associated with the profiles. For quick reference, the basic difference is the bitrates: (TODO). The other option that is used with prores is the -pix_fmt option. This is normally set to yuv422p10le or something like that, but if you want to use the 4444 prores you would set it to yuva444p10le. (A list of possible pixel formats can be invoked by running ffmpeg -pix_fmts. Note that not all of these formats are actually supported with prores). " etc.

**Comment by Nico-Duduf** [comment]
Yup, but I have to check how Dume handles pixformat according to these choices.
Anyway, I'll fix that in 0.4.0 soon!

**Comment by hansnolte** [comment]
Great,
I'm looking forward to the 0.4.0.

Thank you Nico!
